src/main/java/koval/myfit/mobile/gui/pages/android/loginPages/AboutMePage.java

```diff
@@ -0,0 +1,58 @@
+package koval.myfit.mobile.gui.pages.android.loginPages;
+
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import koval.carinademo.mobile.gui.pages.service.interfaces.IConstantUtils;
+import koval.myfit.mobile.gui.pages.common.loginPages.AboutMePageBase;
+import koval.myfit.mobile.gui.pages.common.loginPages.TrackActivitiesPageBase;
+import koval.myfit.mobile.gui.pages.service.enums.LoginPagesTitles;
+import koval.myfit.mobile.gui.pages.service.enums.PersonCharacteristics;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = AboutMePageBase.class)
+public class AboutMePage extends AboutMePageBase implements IMobileUtils, IConstantUtils {
```

Comment:
Please create AbstractPage with implements IMobileUtils, IConstantUtils. and extends it each PageBase. 
you will avoid duplicating (implements IMobileUtils, IConstantUtils)